routers/web/webfinger.go

```diff
@@ -86,6 +87,7 @@ func WebfingerQuery(ctx *context.Context) {
 
 	aliases := []string{
 		u.HTMLURL(),
+		appURL.String() + "api/v1/activitypub/user/" + strings.ToLower(u.Name),
```

Comment:
```suggestion
		appURL.String() + "api/v1/activitypub/user/" + url.PathEscape(u.Name),
```

Reply:
> I think `strings.ToLower` is still required?

Handling username case is a very tricky issue and I've already encountered a Mastodon <-> Gitea federation bug due to Gitea considering Ta180m and ta180m to be the same user while Mastodon thinks they are two different users. I think the best way forward is for Gitea to only use the original case version of the username for federation so other AP software don't get confused.